src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int TABLE_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private Node[] table = new Node[DEFAULT_CAPACITY];
+
+    public static class Node<K, V> {
```

Comment:
```suggestion
    privare static class Node<K, V> {
```

I think we could make this class private